tests/test_templater.py

```diff
@@ -327,3 +325,13 @@ def test_set_template_verbosity(): #pylint: disable=unused-variable
     outcome = ''
     # Check that only the correct messages were logged
     assert result == outcome
+
+def test_mutually_exclusive_args(): #pylint: disable=unused-variable
+    ''' Test that -q, -v, -d args are mutually exclusive'''
```

Comment:
Here, you're suggesting that `-q`, `-v`, and `-d` should be mutually exclusive, but you only included `-q` and `-v` here in the test. They were also the only two included in the `group` addition in the `scripts` files.

Should the comment just include `-q` and `-v`?

Reply:
Comment updated in test. -q and -v should be mutually exclusive.